spec/index.bs

```diff
@@ -11481,8 +11545,59 @@ called the render pass encoder can no longer be used.
             <div data-timeline=queue>
                 [=Queue timeline=] steps:
 
-                1. Issue: Perform the attachment stores/discards.
+                1. For each non-`null` |colorAttachment| in |renderState|.{{RenderState/[[colorAttachments]]}}:
+                    1. Let |colorSubresource| be |colorAttachment|.{{GPURenderPassColorAttachment/depthSlice}} of
```

Comment:
same comment about "subresource" and "depthSlice"

Reply:
Done.